sdk/storage/Azure.Storage.Files/tests/DirectoryClientTests.cs

```diff
@@ -130,7 +130,7 @@ public async Task CreateAsync_SmbProperties()
                 var smbProperties = new FileSmbProperties
                 {
                     //TODO FilePermissionKey
-                    FileAttributes = NtfsAttributes.Directory | NtfsAttributes.ReadOnly,
+                    //FileAttributes = FileNtfsAttributes.Parse("Directory|ReadOnly"),
```

Comment:
Why is this being commented out?

Reply:
An oversight on my part, I'll fix in the next iteration.

Reply:
fixed.